jest-preset.js

```diff
@@ -44,4 +44,8 @@ module.exports = {
     '/node_modules.*/',
     '<rootDir>/(coverage|dist|lib|tmp).*/',
   ],
+  reporters: [
+    'default',
+    '<rootDir>/node_modules/skuba/lib/cli/test/reporters/github',
```

Comment:
How about we `require.resolve` this path? That should allow for internal use, though you'd still have to build `skuba` itself beforehand. It should also serve as an integration test of sorts as I assume our CI will break if we fail to resolve this reporter.